.changeset/breezy-forks-share.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+Remove unnecessery expect
```

Comment:
The description of the changeset does not have enough information to know what has been changed,